src/components/accordion/_accordion.scss

```diff
@@ -153,17 +153,18 @@
 
   .#{$prefix}--accordion__item {
     transition: all $transition--base $carbon--standard-easing;
-    border-top: 1px solid $ui-03;
```

Comment:
Would it be too much of a stretch to keep colors where they are like you had them for default theme use, and then you could use the `accordion--color` mixin to overwrite those colors to optimize the code a little more?

Reply:
But then it's not DRY... we'd have to always remember to include color overrides. Could be easier than having to split up declarations like `border-top` though... so ya, I'd support that as a viable option.